create_coda_files/Pipfile

```diff
@@ -0,0 +1,13 @@
+[[source]]
+url = "https://pypi.org/simple"
+verify_ssl = true
+name = "pypi"
+
+[packages]
+coredatamodules = {editable = true, ref = "v0.7.0", git = "https://www.github.com/AfricasVoices/CoreDataModules"}
```

Comment:
Fine with this being 0.7.0 for now, but you should to check the release notes for 0.7.1 and 0.7.2 and consider upgrading before you interact with coda, particularly when importing back from coded coda files.

Reply:
Noted, upgrading